bundles/persistence/org.openhab.persistence.jdbc/java/org/openhab/persistence/jdbc/db/JdbcMariadbDAO.java

```diff
@@ -67,6 +66,18 @@ private void initDbProps() {
         databaseProps.setProperty("minimumIdle", "2");
     }
 
+    @Override
+    public void initAfterFirstDbConnection() {
+        logger.debug("JDBC::initAfterFirstDbConnection: Initializing step, after db is connected.");
+        dbMeta = new DbMetaData();
+        // Initialize sqlTypes, depending on DB version for example
+        if (dbMeta.isDbVersionGreater(5, 1)) {
```

Comment:
Maybe should check for >= 5.6.4 if [this](https://dev.mysql.com/doc/refman/5.6/en/date-and-time-functions.html#function_now) is correct?